mappings/net/minecraft/server/command/CommandManager.mapping

```diff
@@ -17,7 +19,18 @@ CLASS net/minecraft/class_2170 net/minecraft/server/command/CommandManager
 		ARG 0 name
 		ARG 1 type
 	METHOD method_9247 literal (Ljava/lang/String;)Lcom/mojang/brigadier/builder/LiteralArgumentBuilder;
+		ARG 0 literal
 	METHOD method_9249 execute (Lnet/minecraft/class_2168;Ljava/lang/String;)I
 		ARG 1 commandSource
 		ARG 2 command
 	CLASS class_2171 CommandParser
+	CLASS class_5364 RegistrationRule
```

Comment:
more like an `Environment` or `EnvironmentType`

Reply:
This is used on a data provider class aside from server setup. Maybe `RegistrationEnvironment`?

Reply:
Went for registration environment